migrations/20230604175742_init.sql

```diff
@@ -0,0 +1,49 @@
+-- Add migration script here
+CREATE TABLE IF NOT EXISTS config (
```

Comment:
I've never written any raw sql statements just yet, so I don't get why you can't add a new column.
Regarding the second point, the bot will not configure as soon as you start it so the config should be complete.
The last point is also due to my lacking knowledge of sql. I thought it would be fine to have a table where only one row exists (the one with the current config). 

Since you cleary have a better understandig of sql, I'm open for any suggestions!

Reply:
To build the whole config I would then have to do 5 requests? Sounds kinda iffy, if we keep the central approach of the config. If we change to a table with [key, value] entries I would suggest to also fracture the config loading.

Reply:
Well if it something like a qr-code which needs non-trivial setup, you have to write code for it anyway. That would then be exactly what you would also put into the migration code, no? Just create it with a sensible default and pick up on initialization in bot code.

I mean I see that it is in general more flexibel with a kv-store, but for the bot I don't see the immediate benefit. Maybe we can postpone this discussion to a later point / create an issues?